api/canon/Microsoft.Quantum.Canon.yml

```diff
@@ -1363,24 +1367,24 @@ operations:
 - uid: microsoft.quantum.canon.with
   summary: >-
     Given operations implementing operators `U` and `V`, performs the
-    operation `U†VU` on a target. That is, this operation
+    operation `U�VU` on a target. That is, this operation
```

Comment:
I believe these shouldn't be part of the change; https://github.com/Microsoft/QuantumLibraries/pull/12/files would fix it if merged.